test/unit/workspace/datastore_test.py

```diff
@@ -234,8 +234,10 @@ class TestZenodoFetcher(unittest.TestCase):
             },
         ]
     }
-    PROD_EPACEMS_DOI = "10.5281/zenodo.10233186"
-    PROD_EPACEMS_ZEN_ID = 10233186  # This is the last numeric part of doi
+    PROD_EPACEMS_DOI = datastore.ZenodoDoiSettings().epacems
+    PROD_EPACEMS_ZEN_ID = re.search(r"[^zendodo.]*$", PROD_EPACEMS_DOI)[
+        0
+    ]  # This is the last numeric part of doi
```

Comment:
this was in response to [this comment](https://github.com/catalyst-cooperative/pudl/pull/3096#discussion_r1418197043). now we don't need to update it at all.